Makefile

```diff
@@ -8,10 +8,10 @@ COMPOSE = docker compose -f docker-compose.yml $$(${COMPOSE_CIRRUS})
 COMPOSE_RUN = ${COMPOSE} run --rm
 COMPOSE_LEGACY = ${COMPOSE} -f docker-compose-legacy.yml
 COMPOSE_TEST = docker compose -f docker-compose-test.yml
-COMPOSE_TEST_RUN = ${COMPOSE_TEST} run --rm
+COMPOSE_TEST_RUN = ${COMPOSE_TEST} run --name experimenter_test
 COMPOSE_PROD = docker compose -f docker-compose-prod.yml $$(${COMPOSE_CIRRUS})
 COMPOSE_INTEGRATION = ${COMPOSE_PROD} -f docker-compose-integration-test.yml $$(${COMPOSE_CIRRUS})
-COMPOSE_INTEGRATION_RUN = ${COMPOSE_INTEGRATION} run --rm
+COMPOSE_INTEGRATION_RUN = ${COMPOSE_INTEGRATION} run --name experimenter_integration
```

Comment:
Wait hollup, the `--rm` at the end there removes the temporary containers that were generated, do we want to remove that?

Reply:
I added a `docker rm` at the end of each of the check/lint rules. It seems to work fine in my testing.

Reply:
Yeah, or else the container is deleted